internal/controller/services/monitoring/monitoring_controller_actions.go

```diff
@@ -95,15 +98,96 @@ func createMonitoringStack(ctx context.Context, rr *odhtypes.ReconciliationReque
 
 	if monitoring.Spec.Metrics != nil && (monitoring.Spec.Metrics.Resources != nil || monitoring.Spec.Metrics.Storage != nil) {
 		if msExists, _ := cluster.HasCRD(ctx, rr.Client, gvk.MonitoringStack); !msExists {
-			return errors.New("MonitoringStack CRD not found")
+			// CRD not available, skip monitoring stack deployment (this is expected when monitoring stack operator is not installed)
+			rr.Conditions.MarkFalse(
+				status.ConditionMonitoringStackAvailable,
+				conditions.WithReason(status.MonitoringStackOperatorMissingReason),
+				conditions.WithMessage(status.MonitoringStackOperatorMissingMessage),
+			)
+			return nil
 		}
+
+		rr.Conditions.MarkTrue(status.ConditionMonitoringStackAvailable)
+
 		template := []odhtypes.TemplateInfo{
 			{
 				FS:   resourcesFS,
 				Path: MonitoringStackTemplate,
 			},
 		}
 		rr.Templates = append(rr.Templates, template...)
+	} else {
+		// No metrics configuration or metrics not sufficiently configured
+		rr.Conditions.MarkFalse(
+			status.ConditionMonitoringStackAvailable,
+			conditions.WithReason(status.UnmanagedReason),
```

Comment:
is this reason correct: 
should be something like, No Metrics configed in DSCI ?

Reply:
you're correct